### PR TITLE
Create Invoices.md

### PR DESCRIPTION
**Pull request by pavlenex**
Ready to be reviewed.
Tackles #237 

Posting it for a review and general brainstorming about what can be added there. Eventually, I plan to create "BTCPay Invoice Management video"

### CODE REVIEW

**Comment by pavlenex** [comment]
This one is now good to go. Will add a video once I make it.

File: Invoices.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+The invoices page lists the invoices from all stores and [apps](apps.md) and [payment requests](http://PaymentRequests.md). It cumulatively sorts invoices by date and allows plenty of customization options for a better invoice management.
```
Comment (britttttk) [inline Invoices.md]: remove http from link :)

File: Invoices.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+| New                    | Not paid, invoice timer still has not expired                | None                                                         |
+| Expired                | Not paid, invoice timer expired                              | None                                                         |
+| Paid                   | Paid, but has not received sufficient amount of confirmations specified in the Store Settings | Wait for confirmations (The invoice should become - complete) |
+| Completed              | Paid, completed, received sufficient amount of confirmations | Fulfil the order                                             |
+| Expired (paidPartial)* | Paid, but not in full amount and expired                     | Contact buyer to arrange a refund or ask for them to pay their due. Optionally mark invoice as completed |
+| Paid (Paid Over)       | Paid more than the invoice amount, but has not received sufficient amount of confirmations specified in the Store Settings | Wait for confirmations. The invoice should become - Complete (Paid Over). Then, |
```
Comment (britttttk) [inline Invoices.md]: The invoice should become - Complete (Paid Over). [Then,]<-- remove

**Comment by pavlenex** [comment]
@britttttk Done let me know if there's anything else to fix.

**Pull request merged**